lib/trento/application/integration/discovery/policies/host_policy.ex

```diff
@@ -95,6 +140,52 @@ defmodule Trento.Integration.Discovery.HostPolicy do
            installation_source: installation_source
          })
 
+  defp build_update_saptune_command(agent_id, nil, package_version, saptune_installed),
```

Comment:
I would put the status as last argument, to follow the same order we have in the command. Just ocd XD

Reply:
done :D